draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -1058,6 +1035,41 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
         
     </section> <!-- Validating Phase -->
 
+    <section anchor="sec-notes-BBR" title="Implementation Notes for using BBR">
+
+        <t>When the flow is controlled using BBR, Careful Resume is implemented
+           by setting the pacing rate from the saved congestion control parameters,
+           with the following precautions:</t>
+           <t><list style="symbols">
+		   <t>The flag "carefully-resuming" is added to the BBR state
+              to indicate that the sender is allowed to send unvalidated
+		      packets.  
+		       It is initialized to "False" when the BBR flow starts;</t>
+		   <t>Careful Resume is allowed to transmit unvalidated packets
+			  only when the BBR flow is in the Startup state;</t>
```

Comment:
Is it useful here to point (ref) to the conditions specified for reconnaissance to show a sender knows to set the flag. This could require something or be a xref reference to say similar checks are needed.